mobile/src/main/java/com/truckmuncher/truckmuncher/App.java

```diff
@@ -19,6 +23,9 @@ public static void inject(Context context, Object target) {
     @Override
     public void onCreate() {
         super.onCreate();
+
+        TwitterAuthConfig authConfig = new TwitterAuthConfig(BuildConfig.TWITTER_API_KEY, BuildConfig.TWITTER_API_SECRET);
+        Fabric.with(this, new Crashlytics(), new Twitter(authConfig));
```

Comment:
I have some concerns about this.

1. Wouldn't this make crashlytics run in debug mode as well? The old Crashlytics didn't "ignore" things in debug, but we definitely don't want them reported. That was why it was in the `if...else` in `LoggerStarter`.
2. The reason for crash logging to be initialized in `LoggerStarter` is b/c if the content provider is accessed through a `Context` which is not ours (like if we export our data), then our Application does not get launched so crashes would not get reported to us. `LoggerStarter` is initialized in the content provider, which circumvents this limitation.
3. Does this create a Twitter singleton that we keep around for the lifetime of our app? If so, that seems like a waste of memory since we only use it for the login stuff and even then only for vendors who actually use twitter.